test/Microsoft.AspNetCore.Mvc.ApiExplorer.Test/DefaultApiDescriptionProviderTest.cs

```diff
@@ -1416,6 +1417,172 @@ public void GetApiDescription_WithControllerProperties_Merges_ParameterDescripti
             Assert.Equal(typeof(string), comments.Type);
         }
 
+        [Fact]
+        public void ProcessIsRequired_SetsTrue_ForFromBodyParameters()
+        {
+            // Arrange
+            var description = new ApiParameterDescription { Source = BindingSource.Body, };
+            var context = GetApiParameterContext(description);
+
+            // Act
+            DefaultApiDescriptionProvider.ProcessIsRequired(context);
+
+            // Assert
+            Assert.True(description.IsRequired);
+        }
+
+        [Fact]
+        public void ProcessIsRequired_SetsTrue_ForParametersWithBindRequired()
```

Comment:
Test is contradictory, description is for a parameter but metadata is for a property.

Reply:
ApiParameters cover parameters and properties

Reply:
🆗 but this test still covers a C# property, making the name confusing. Suggest changing the name to `..._ForApiParameterWithBindRequired` or `..._ForPropertyWithBindRequired`.